data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,283 @@ mission "Arachi Cartoonist 2"
+			`	"In many ways we differ, when it comes to physical form," Pairri says. "So in the cooperative match, when come together as one we do, different tasks, each member of the team is assigned. For the sake of fairness, it is so."`
+			choice
+				`	"What are they carrying in those bags?"`
+				`	"So is pulling those chariots the task of the Saryds?"`
+			`	A loud airhorn interrupts Pairri as she is about to answer, and the Saryds start galloping toward the wall. The Kimek and Arachi in each chariot do their best to stay on, as their Saryd teammates seem to generally press on with little regard for the chariot's stability; instead they are focused solely on reaching the wall as fast as possible.`
+			`	It's a very tight race, but the Saryds begin slowing once they approach the wall, both out of exhaustion and to prevent a crash. As the Saryds arrive at the base of the cliff, they immediately lay down to rest while the Arach on their team begins to unstrap the chariot. The Kimek hurriedly begins climbing the wall with the bags they had packed earlier.`
+			`	Before long, the Kimek have all reached the top of the cliff and begin opening their bags to grab a multitude of paddles, some kind of pole which looks vaguely like a combination shovel and a pickaxe, and a long rope. "To cross the river, the primary goal of the game is," Pairri explains. "Prepare the raft for their team, the Kimek must. My role this would have been. As skilled in anticipating the actions of my friends I hope my former opponent is."`
+			`	Back at the base of the cliff, the Arachi are done helping their Saryd teammates out of the chariot straps, and together they prepare for the climb themselves. Shockingly, the Saryds lay across the smaller Arachi's backs, and together they use the lengths to fabric to secure themselves to each other. It seems that the Arachi use these straps to carry their larger teammates up the cliff. You have your monitor focus on Pairri's friends again, and see that Ulmodi is just finishing checking his safety straps, after which Tumub begins his climb. Despite the stress of carrying someone much larger than him, Tumub's movements remain precise and deliberate, never more than one leg leaving the wall at a time as he carefully chooses the best spots to anchor himself and support the weight of his Saryd friend.`
+			`	Once Tumub reaches the top, Ulmodi dislodges himself and removes the harness from Tumub. He then lifts his Arach teammate into his arms so that he may climb onto the Saryd's back this time, while moving to the by now nearly complete raft. Tumub climbs from Ulmodi's back into the raft, and Ulmodi helps his Kimek teammate finish properly securing the ropes. Once done, Ulmodi and the Kimek begin pushing the raft to the river, where Ulmodi now lifts up the Kimek and leaps into the raft just after it begins to float on the water.`
```

Comment:
```suggestion
			`	Once Tumub reaches the top, Ulmodi dislodges himself and removes his harness. He then lifts his Arach teammate into his arms so that he may climb onto the Saryd's back this time, while moving to the by now nearly complete raft. Tumub climbs from Ulmodi's back into the raft, and Ulmodi helps his Kimek teammate finish properly securing the ropes. Once done, Ulmodi and the Kimek begin pushing the raft to the river, where he now lifts up the Kimek and leaps into the raft just after it begins to float on the water.`
```